any/attributeFromLargestArea.md

```diff
@@ -0,0 +1,62 @@
+# Attribute from Intersection
+
+This expression returns an attribute value from a featureSet where the largest feature intersects the input feature.
+
+## Use cases
+
+To suggest a value from a majority intersection. For instance, when drawing a polygon, return the county name for which the majority of the new polygon is in.
+
+## Workflow
+
+Copy and paste the expression found in the expression template below to the Arcade editor in ArcGIS Online, the relevant location in ArcGIS Pro, or the relevant location in a custom app.
+
+To configure the script to your layer, edit the first line to specify the layer you would like to use instead of the example layer. 
+
+```js
+var set = FeatureSetByName($datastore,"Building Footprints")
+```
+
+Also edit the last line to specify the attribute you would like to use intead of the example `objectid`.
+
+```js
+getAttributeFromLargestArea($feature, set, 'objectid')
+```
+
+## Expression Template
+
+This Arcade expression will extract the attribute value from a feature in a featureSet where the input feature intersection area is the largest.
+
+```js
+var set = FeatureSetByName($datastore, 'Building Footprints')
+
+function getAttributeFromLargestArea(feat, set, field) {
+    var items = intersects(set, feat)
+    console(items)
+
+    if (items == false) {
```

Comment:
I believe this will always return `false` so the error will never be returned. Maybe do a `(count(items) < 1)` or `(count(items) == 0)` instead?

Reply:
This is not what happens from my experience. If there is no intersection then items is a boolean `false` not an empty list. So checking for `count(items)` when it is 0 throws an error.

Reply:
That's odd. When a featuerset is used in a geometry function it should always return a featureset. When not used, it returns a boolean. I tried this in the Playground: https://developers.arcgis.com/arcade/playground/

```
var set = intersects(FeatureSetByName($datastore,"Tax Parcels"), $feature);
set == false
// returns false because set contains one feature.

var set = crosses(FeatureSetByName($datastore,"Tax Parcels"), $feature);
set == false
// returns false because set is an empty featureset
```

If your data is giving something different, then maybe we have a bug somewhere.

Reply:
Weird, this was happening in the playground but is not happening today.

Reply:
i switched to count == 0 but you closed the pr before so if you want to update that please do. cherry pick https://github.com/steveoh/arcade-expressions/commit/bf4313f38161b3e76c0f1da1c3e3ee5ed1c0f322

Reply:
Gah. Sorry about that. I'll update it.

Reply:
then add the semi-colon or cherry pick c875a8e83f60663c818436e81823c2d5f4c1f678